api/src/pcapi/core/testing.py

```diff
@@ -271,3 +275,48 @@ def disable(self):
         if flask.has_request_context():
             if hasattr(flask.request, "_cached_features"):
                 flask.request._cached_features = {}
+
+
+def clean_temporary_files(test_function):
```

Comment:
Est-ce que l'on ne pourrait pas utiliser [la fixture pytest pour les répertoires temporaires](https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture) ?

Reply:
Je ne suis pas sûr de comprendre comment on pourrait l'utiliser. Pour moi, cette fixture pourrait être utilisée si l'implémentation prenait en argument un répertoire (qui serait alors créé et nettoyé par la fixture, lors du test). Mais ça n'est pas le cas, et je trouve que ça rendrait le code un peu moche. Je loupe quelque chose ?

Reply:
ah, en effet. Il me semblait que ça faisait aussi les fichiers temporaires en patchant la lib python de base.
Désolé pour le bruit.